folly/synchronization/test/RWSpinLockTest.cpp

```diff
@@ -152,7 +152,8 @@ TYPED_TEST(RWSpinLockTest, ConcurrentTests) {
   srand(time(nullptr));
 
   std::vector<std::thread> threads;
-  for (int i = 0; i < FLAGS_num_threads; ++i) {
+  threads.reserve(FLAGS_num_threads);
+for (int i = 0; i < FLAGS_num_threads; ++i) {
```

Comment:
Bad formatting. And why bother here?

Reply:
I was mixed on the vector reserve checks. It is nearly no speed-up given the few numbers of `push_back` call in unit tests, typically. Happy to revert.

As for the formatting -- is it possible to get a `.clang-format` in the open source repo?

Reply:
I intentionally didn't apply the vector-reserve or pass-by-const-ref passes when I ran clang-tidy over Folly a while back. The vast majority of the places where the vector-reserve pass changed things were in unit tests, where the added complexity (and in many cases, added duplication of constants), are not desired. I chose not to apply the pass-by-const-ref pass in the vast majority of places because it tried to change multiple places that I knew would break things.

Reply:
Thanks for the insights, @orvid. I wasn't aware someone had already run clang-tidy over the entire codebase.

I agree on both fronts with the vector-reserve pass and the pass-by-const-ref pass after looking at things a bit closer.